src/toast/instrument_sim.py

```diff
@@ -577,13 +634,15 @@ def fake_hexagon_focalplane(
 
     nominal_freq = str(int(bandcenter.to_value(u.GHz)))
     det_names = [f"{x}-{nominal_freq}" for x in det_data.keys()]
+    det_gamma = [dgamma[x] for x in det_data.keys()]
```

Comment:
we need to propagate this change to all  Instrument Models (IMO)  that use toast3. What happens if an IMO doesn't encode that ? toast should raise an error, shouldn't it ?

Reply:
There is an exception raised if the `hwp_angle` is not `None` in the StokesWeights operator and the focalplane is missing the `gamma` key.